.travis.yml

```diff
@@ -14,6 +14,7 @@ script:
 env:
     # This is used to only capture the regular nightly test in allow_failures
     - REGULAR_TEST=true
+    - PUBLISH_DOC=true
```

Comment:
Could we inverse the logic here by setting the variable to true at one matrix item, and leaving it unspecified otherwise? This was we can mark what where we want to publish the docs more clearly. In other words, change this variable to false and mark the build that must publish the docs as true.

Reply:
Done :)